packages/components/src/molecules/Card/Card.tsx

```diff
@@ -0,0 +1,58 @@
+import type { HTMLAttributes } from "react";
+import React, { forwardRef } from 'react'
+
+import { Icon } from '../..'
+
+export interface CardProps extends Omit<HTMLAttributes<HTMLDivElement>, 'role'> {
+  /**
+  * Specifies the text to be loaded into the header.
+  */
+  title?: string
+  /**
+  * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+  */
+  testId?: string
+  /**
+  * Specifies the card max width.
+  */
+  maxWidth?: string
+  /**
+  * Specifies what icon to use following the FastStore Icon sheet.
+  */
+  iconName?: string
+  /**
+  * Specifies the icon's color
+  */
+  iconColor?: string
+  /**
+  * Function that should be executed when the icon is clicked
+  */
+  iconAction?: () => void
+}
+
+const Card = forwardRef<HTMLDivElement, CardProps>(function Card({
+  title,
+  maxWidth,
+  testId = 'fs-card',
+  iconName,
+  iconColor,
+  iconAction,
+  children,
+  ...otherProps
+}, ref) {
+  return (
+    <section data-testid={testId} ref={ref} data-fs-card style={{ maxWidth: `${maxWidth}` }} {...otherProps}>
+      <div data-fs-card-header>
+        <div data-fs-card-title>{title}</div>
+        {iconName && (
+          <Icon name={iconName} width={20} height={20} weight="bold" color={iconColor} onClick={iconAction} />
```

Comment:
We usually have a data-attr for icon [customization](https://github.com/vtex/faststore/blob/ff12ccfdf1cc909dc708912fa5b0c95e6aaaef57/packages/ui/src/components/molecules/Gift/styles.scss#L24), for example `data-fs-card-icon` and  a `--fs-card-icon-color` token to change the icon color. 

cc: @renatamottam

Reply:
Changed it